store/sqlite.go

```diff
@@ -296,6 +316,30 @@ func (db *DB) migrateFile(name string, up bool) error {
 	return tx.Commit()
 }
 
+// pgMigrateFile runs a migration file within a transaction.
+func (db *DB) pgMigrateFile(name string, up bool) error {
+	if up {
```

Comment:
I suggest to only provide up migration. I rarely see cases we need to rollback (roll forward is always a better option)

Reply:
Per offline discussion, do it in a separate PR